.github/workflows/manpage-build.yml

```diff
@@ -0,0 +1,22 @@
+name: Pages
+on:
+  push:
+    branches:
+    - main
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
```

Comment:
Just a tip: I find it much more readable if you insert a linebreak between each step. This makes each step much more readable as it is shown as one unit.